loadtests/artipie-snapshot.sh

```diff
@@ -6,9 +6,10 @@ fi
 port="$1"
 JMX_PORT=9999
 JDEBUG=8000
+ARTIPIE_JVM_EXTRA="-XX:StartFlightRecording=disk=true,filename=/var/artipie/.storage/data/chgen/bintest/artipie.jfr"
```

Comment:
How is planned get `artipie.jfr` for studying?

Reply:
@dgarus , it will be downloaded from artipie with wget, from GitHub action scripts.